src/BuiltInTools/BrowserRefresh/HostingFilter.cs

```diff
@@ -23,6 +23,12 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
         {
             return app =>
             {
+                app.Map("/_framework/clear-browser-cache", app1 => app1.Run(context =>
```

Comment:
I assume this behavior will be sticking around for a while. If so, might help to add a comment on why we do this here (AKA move the comment below into the code).